yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -1025,9 +1025,6 @@ object Client extends Logging {
         "future version of Spark. Use ./bin/spark-submit with \"--master yarn\"")
     }
 
-    // Set an env variable indicating we are running in YARN mode.
-    // Note that any env variable with the SPARK_ prefix gets propagated to all (remote) processes
-    System.setProperty("SPARK_YARN_MODE", "true")
```

Comment:
Ok, to be safe I will revert the last commit where I removed the other
places where it was being set and add a comment there

On Tuesday, October 20, 2015, Imran Rashid <notifications@github.com> wrote:

> In yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala
> <https://github.com/apache/spark/pull/9174#discussion_r42515407>:
>
> > @@ -1025,9 +1025,6 @@ object Client extends Logging {
> >          "future version of Spark. Use ./bin/spark-submit with \"--master yarn\"")
> >      }
> >
> > -    // Set an env variable indicating we are running in YARN mode.
> > -    // Note that any env variable with the SPARK_ prefix gets propagated to all (remote) processes
> > -    System.setProperty("SPARK_YARN_MODE", "true")
>
> to @tgravescs <https://github.com/tgravescs> point, would it hurt to
> leave this in here? and then change the comment to indicate that it should
> normally be set by SparkSubmit but left here in case anybody is still using
> this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/9174/files#r42515407>.
>


-- 

Thanks,
Hari